.github/workflows/prepare-release-branch.yml

```diff
@@ -0,0 +1,177 @@
+name: Prepare release branch
+on:
+  workflow_dispatch:
+    inputs:
+      prerelease_version:
+        description: "Pre-release version number? (e.g. 1.9.0rc2)"
+        required: false
+
+jobs:
+  prereqs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Verify prerequisites
+        env:
+          PRERELEASE_VERSION: ${{ github.event.inputs.prerelease_version }}
+        run: |
+          if [[ $GITHUB_REF_NAME != main ]]; then
```

Comment:
nit:

```suggestion
          if [[ $GITHUB_REF_NAME != ${{ github.event.repository.default_branch }}
 ]]; then
```

Reply:
there are several places that refer to "main", I'm not sure it makes things clearer to change all references from "main" to "the default branch"

Reply:
I was trying this on my fork with different default branch. So it was not successful at the start. I am fine with keeping it main.